src/native/libs/System.Native/pal_uid.h

```diff
@@ -82,3 +82,17 @@ PALEXPORT int32_t SystemNative_GetGroupList(const char* name, uint32_t group, ui
 * If the buffer is too small, errno is EINVAL.
 */
 PALEXPORT int32_t SystemNative_GetGroups(int32_t ngroups, uint32_t* groups);
+
+/**
+* Gets the user name associated with the specified user ID and stores it in the buffer.
```

Comment:
(nit) It might also be good to document whether the returned `char*` should be free'd by the caller or not.

Sometimes methods return shared data that shouldn't be free'd, sometimes they return new memory (like in this case).

Reply:
From the answer in your other comment, it will get freed automatically, as that is the expected/default behavior in string marshalling when you return a string.

Reply:
I think it would still be good to call out ownership / that it returns new memory. It would be clearer for consumers that relying on the marshalling behaviour of freeing the string would be correct.

Reply:
Done.